api/src/main/java/module-info.java

```diff
@@ -270,7 +272,8 @@
  *
  * </table>
  *
- * <p>A Jakarta Data provider might allow additional entity attribute types.</p>
+ * <p>All of the basic types are sortable except for {@code byte[]}.
+ * A Jakarta Data provider might allow additional entity attribute types.</p>
```

Comment:
I added this sentence so that we can concisely refer to "sortable basic types" in the table.